python/doc/theory/data_analysis/quantile_estimation_wilks.rst

```diff
@@ -1,197 +1,90 @@
-+-------------+------------------------------------------+--------------------------------------+
-| 832         | 801                                      | 791                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 855         | 823                                      | 813                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 877         | 844                                      | 834                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 900         | 866                                      | 856                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 923         | 888                                      | 877                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 945         | 909                                      | 898                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 968         | 931                                      | 920                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 991         | 953                                      | 942                                  |
-+-------------+------------------------------------------+--------------------------------------+
-
-:math:`\widehat{q}_{Y^i}(\alpha)` is often called the “empirical
-:math:`\alpha`-quantile” for the variable :math:`{Y^i}`.
+    k_{sol} & = \min \{ k \in [1, n] \, | \, F_{\sampleSize, \alpha}(k-1)\geq \beta \}\\
+            & = 1 +  \min \{ k \in [1, n] \, | \, F_{\sampleSize, \alpha}(k)\geq \beta \}
+
+An upper bound of  :math:`x_{\alpha}` is estimated by the value of :math:`X_{(k_{sol})}`  on the sample
+:math:`(x_1, \dots, x_\sampleSize)`.
+
+Estimation of the smallest sample size that ensures that a given order statistics will provide an upper bound
+to :math:`x_{\alpha}`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~
+
+Given :math:`\alpha`, :math:`\beta`, and :math:`k`, we seek for the smallest sample size :math:`\sampleSize`
+that verifies :eq:`EqOrderStat`. This time, :eq:`EqOrderStat2` is solved wrt :math:`\sampleSize`.
+
+Once the smallest size :math:`\sampleSize`  has been estimated, a sample of size :math:`\sampleSize` can be
+generated from
+:math:`X` and an upper bound of :math:`x_{\alpha}` is estimated by the value of the value of
+:math:`X_{(\sampleSize-i)}` on the sample
+:math:`(x_1, \dots, x_\sampleSize)`.
```

Comment:
The observation of $X_{(n - i)}$ on the sample $(x_1, \dots, x_n)$ is ... $x_{(n - i)}$:
```suggestion

Once the smallest size :math:`\sampleSize`  has been estimated, a sample of size :math:`\sampleSize` can be
generated from
:math:`X` and an upper bound of :math:`x_{\alpha}` is estimated using :math:`x_{(\sampleSize - i)}`, i.e. the 
:math:`\sampleSize - i`-th observation in the ordered sample :math:`(x_{(1)}, \dots, x_{(\sampleSize)})`.
```

Reply:
yes